exchanges/bithumb/bithumb_wrapper.go

```diff
@@ -122,8 +123,93 @@ func (b *Bithumb) UpdateOrderbook(p pair.CurrencyPair, assetType string) (orderb
 // GetAccountInfo retrieves balances for all enabled currencies for the
 // Bithumb exchange
 func (b *Bithumb) GetAccountInfo() (exchange.AccountInfo, error) {
-	var response exchange.AccountInfo
-	return response, errors.New("not implemented")
+	var info exchange.AccountInfo
+	info.ExchangeName = b.GetName()
+
+	bal, err := b.GetAccountBalance("ALL")
+	if err != nil {
+		return info, err
+	}
+
+	var exchangeBalances []exchange.AccountCurrencyInfo
+
+	// Added due to increasing of the usuable currencies on exchange, usually
+	// without notificatation, so we dont need to update structs later on
+	for tag, datum := range bal.Data {
+		splitTag := common.SplitStrings(tag, "_")
+		c := splitTag[len(splitTag)-1]
+		switch splitTag[0] {
+		case "available":
```

Comment:
They are continuously adding new cryptocurrency to this end point, so this will expand to available currencies when there is an addition without the need to change structs. But I see your point. I use those cases to throw away unneeded variables. I can populate the entirety to a struct on the api level so devs who use this can get full utilisation of endpoint but we only take what we need?

Reply:
Yeah pretty much. I like what you;ve done regarding the generic struct style to handle all the currency additions. The specific api implementation would be on  bithumb.go and only the stuff we care about like "in" and "total" would be considered at the wrapper.

Reply:
okay will change 👍